JitsiConference.js

```diff
@@ -91,14 +106,57 @@ function JitsiConference(options) {
     /**
      * Indicates whether the connection is interrupted or not.
      */
-    this.connectionIsInterrupted = false;
+    this.isJvbConnectionIsInterrupted = false;
+
+    /* P2P related fields below: */
+
+    /**
+     * Stores reference to deferred start P2P task. It's created when 3rd
+     * participant leaves the room in order to avoid ping pong effect (it
+     * could be just a page reload).
+     * @type {number|null}
+     */
+    this.deferredStartP2P = null;
```

Comment:
None of the fields of any class are really public. It's just the public/private doc (and _ ) were not adopted yet and I don't want to make confusion here with one or two fields. All of them should be converted at once.